automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -50,13 +50,14 @@ def satellite6_upgrade(admin_password=None):
         e.g '6.1','6.2'
     """
     to_version = os.environ.get('TO_VERSION')
+    sat_host = os.environ.get('SAT_HOST')
```

Comment:
Can we rename `sat_host` to `rhev_sat_host` just to differentiate between user provided Satellite Host and Host created from RHEVM image ?